firmware/atom_s3_i2c_display/README.md

```diff
@@ -46,3 +46,65 @@ If you want to set the I2C address, set the `I2C_ADDR` environment variable to t
 ```
 I2C_ADDR=0x42 pio run -t upload
 ```
+
+## Control multiple modes
+
+The `atom_s3_i2c_display` includes multiple modes. This feature allows the robot to be controlled using both the `/i2c_button_state` and `/i2c_mode topics`, making control programming easier.
+
+### Program Structure
+
+- The AtomS3 firmware is paired with a corresponding rospy program on the computer. To use any mode, first update the AtomS3 firmware, then launch the matching rospy node.
+
+| AtomS3 Firmware | Computer rospy Node |
+| ------------- | ------------- |
+| display_information_mode.cpp | display_information.py |
+| display_qr_code_mode.cpp | display_information.py |
+| servo_control_mode.cpp | servo_control_mode.py |
+| pressure_control_mode.cpp | pressure_control_mode.py |
+| teaching_mode.cpp | teaching_mode.py |
+
+- `i2c_button_state_publisher.cpp` facilitates communication between these programs.
+The following sections provide details on the base code for the AtomS3 firmware.
+
+- `main.cpp` includes instances for each mode and stores them in a list. Each time a long click occurs, the active task switches to another mode.
```

Comment:
It would be good to note that it does not include instances of Servo Control Mode, Pressure Control Mode and Teachiing Mode by default.